### ISSUE TITLE
Should max-age have `=` in express?

### ISSUE THREAD

**Issue by webuniverseio** [sdgluck/cache-control#4]
Whey you use max-age, code that is generated in express shows following:

`res.set('Cache-Control', 'max-age 86400');`

shouldn't there be `=` instead of space, like so

`res.set('Cache-Control', 'max-age=86400');`

Thank you

**Comment by sdgluck** [comment]
Yes appears you're right! Thanks for letting me know will fix asap.

**Comment by abcfy2** [comment]
Still not fix yet?

**Comment by FINDarkside** [comment]
I thought I was smart by using builder instead of writing my Cache-Control manually to avoid small typos and wasted time. But wasted a lot time because of this :laughing: In case it's not obvious, this isn't anything express specific, this tool simply does not output valid Cache-Control headers.

**Comment by sdgluck** [comment]
Thanks for letting me know, and sorry about that @FINDarkside. I will take it down as I don't have time to maintain this anymore. All the best.
